### PR TITLE
Hyperskill - new logo addition

### LINKED ISSUE #10182: Hyperskill
### Brand Name

Hyperskill

### Website

https://hyperskill.org/

### Popularity Metric

[Hyperskill](https://hyperskill.org/) is a project-based learning platform where you can learn to program by creating real-world applications. In particular, you can study Java, Kotlin, Python, Frontend, Go, Scala, SQL, machine learning, and other subjects.

It is part of [Jetbrains Academy](https://hi.hyperskill.org/about). 

It's Similarweb rank is https://www.similarweb.com/website/hyperskill.org/#overview. 

### Official Resources for Icon and Color

https://assets-global.website-files.com/65255f01672fcf3029794f73/65255f01672fcf3029794f8e_Hyperskill%20Logo.svg

### Additional Comments

Hyperskill has helped me understand concepts in Java, Python and SQl. I think it has highly curated topics which cannot be found elsewhere. 
You can study Java, Kotlin, Python, Frontend, Go, Scala, SQL, machine learning, and other subjects here. I think people would like to showcase their learnings using Hyperskill account.

### PR DESCRIPTION
**Pull request by durgeshgowdac**
ISSUE: Closes #10182 

### 1. Added New Icon:
- Added the **hyperskill.svg** icon to the **simple-icons/icons** directory.

### 2. Updated JSON Data:
- Updated the JSON data in **_data/simple-icons.json** to include information about the new **hyperskill icon**.

</br>

### Icon Details:

* The hyperskill.svg icon represents the [Hyperskill](https://hyperskill.org/) brand.
- The icon follows the standard format and guidelines used in Simple Icons.

### Testing:
- I have tested the icon locally to ensure it displays correctly.

### Visual Inspection:
- Visually inspected the icon to confirm that it aligns with the Simple Icons style and quality.

### CODE REVIEW

**Comment by durgeshgowdac** [comment]
### changed “” to "".
- Encountered a quotation mark mismatch in the JSON file. It might be due to different encoding or formatting issues.

**Comment by durgeshgowdac** [comment]
Changed lowercase to uppercase in HEX

**Comment by durgeshgowdac** [comment]
## Pull Request Summary
This pull request adds Hyperskill Logo.

### Changes Made
- Added the hyperskill.svg icon to the simple-icons/icons directory.
- Updated the JSON data in _data/simple-icons.json to include information about the new hyperskill icon.

### Issues while commiting
- There was a quotaion mismatch due to different encoding or formatting issue in JSON file.
- Had used lowercase for HEX.

### Resolved Issues
- Changed the quotaions to match the encodings.
- Changed the lowercase HEX to Uppercase HEX.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for the contribution, @durgeshgowdac 👍🏻 However, I'm seeing [some significant differences](https://github.com/simple-icons/simple-icons/assets/15157491/54e43bc0-68fd-4053-bf4e-90b08dd9be63) between your version (in red) and the [logo](https://assets-global.website-files.com/6522a1d9333be21b9e0bf97c/652933d887f235a01afae283_Logo-goal.svg) in their website's header. I notice, though, you based your version on a different file. Whereabouts on their site did you find that?

Also, for the colour, I think I'd expect a darker blue based on the website's colour scheme. Which one, I don't know, though. Perhaps `#5d72e9`?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6634,6 +6634,11 @@
             "hex": "000000",
             "source": "https://hyper.is"
         },
+        {
+            "title": "Hyperskill",
+            "hex": "69A5F7",
+            "source": "https://assets-global.website-files.com/646bbfc4c8c88c7c71128456/646f959078117670eb2a9fd8_Hyperskill%20Logo.svg"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:6640]:
```suggestion
            "source": "https://hyperskill.org"
```

**Review by PeterShaggyNoble — APPROVED** [review]
Your new SVG lines up perfectly with the source, @durgeshgowdac, nice work.

Before I merge, though, what do you think of my suggested change to the colour?

**Comment by durgeshgowdac** [comment]
image

Closes Issue https://github.com/simple-icons/simple-icons/issues/10182

Changes:
- Previously uploaded hyperskill.svg had scaling issue. I've resolved it and uploaded the new optimized hyperskill.svg
- I think Cornflower Blue `#6962F2` is a more dominant color. I've updated the Hex in simpleicons.json

**Comment by durgeshgowdac** [comment]
> @durgeshgowdac, where did you source that new colour from? I'm not seeing it in their website's stylesheet.

I found an extension (site palette) which extracts color palette from the given website. It scanned the whole website and showed `#6962F2` to be the most dominant color.

**Comment by PeterShaggyNoble** [comment]
The problem is, though, @durgeshgowdac, that that colour doesn't appear in their CSS varaiables 🤔

**Comment by durgeshgowdac** [comment]
@PeterShaggyNoble Alright, I will change it back to #5D72E9

**Comment by durgeshgowdac** [comment]
@PeterShaggyNoble Can I change it to `#6C63FF`?

**Comment by PeterShaggyNoble** [comment]
From the variable names above, @durgeshgowdac, it looks to me like that colour is no longer in use.

**Comment by durgeshgowdac** [comment]
> From the variable names above, @durgeshgowdac, it looks to me like that colour is no longer in use.

@PeterShaggyNoble What about `#8C5AFF`? 

If you have any other color in mind. Let me Know

**Comment by PeterShaggyNoble** [comment]
👍🏻 That seems as good a choice as any to me; equally as valid as my suggestion.

**Comment by durgeshgowdac** [comment]
@PeterShaggyNoble I've made all the changes. You can merge now. 

Thank You,
@durgeshgowdac

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**